server/pkg/controller/billing.go

```diff
@@ -53,6 +55,7 @@ func NewBillingController(
 	usageRepo *repo.UsageRepository,
 	storageBonusRepo *storagebonus.Repository,
 	commonBillCtrl *commonbilling.Controller,
+	notificationHistory *repo.NotificationHistoryRepository,
```

Comment:
nit: notificationHistoryRepo or just notificaitonRepo. Please rename it at other places too.